src/js/utils.js

```diff
@@ -17,12 +17,28 @@ exports.parseQueryParams = () => {
 /** Creates an object based on a form's current input values. */
 exports.parseForm = form => {
   const formData = new FormData(form);
-  return Array.from(formData.entries()).reduce((acc, [key, value]) => {
-    if (!acc[key]) {
-      acc[key] = [];
+  if (!formData.entries) {
+    // for IE
+    let filter = {};
+    const elements = document.getElementsByTagName("input");
```

Comment:
You could offload some later work to the browser by using a selector here

```
const elements = document.querySelectorAll('input[type="checkbox"]:checked');
```

Reply:
That's a good idea, done.